.github/workflows/update-core-example.yml

```diff
@@ -0,0 +1,19 @@
+name: Update ChainBridge core-example
+
+on:
+  push:
+    branches:
+      - 'test-cd'
+
+jobs:
+  update-core-example:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger chainbridge-core-example update event
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # Dispatch event from current repository
+          repository: github.repository
+          event-type: update-core-example
```

Comment:
i would call it something like 'push-to-core-main-branch' to make it clear that this event is about new code on main branch in core repo